src/main.h

```diff
@@ -39,6 +40,16 @@ extern "C" {
 
 double tarantool_uptime(void);
 
+/**
+ * Path to Tarantool binary of the process.
+ */
+extern char tarantool_path[PATH_MAX];
+
+/**
+ * Application start time on monotonic clocks.
+ */
+extern time_t start_time_safe;
```

Comment:
Nit: `_safe` suffix looks confusing. Also, it'd be nice to have the `tarantool_` prefix for consistency with `tarantool_path`. Let's rename to `tarantool_start_time`?

Reply:
Done.